Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -368,6 +371,24 @@ def download_artifact(url, dir, pr)
     end
   end
 
+  def pr_check_conflicts(name, repo, pr)
+    long_build_pr_files = GitHub.get_long_build_pull_requests_files(name, repo)
+    this_pr_files = GitHub.get_pr_files(name, repo, pr)
+
+    conflicts = []
+    this_pr_files.each do |this_pr_file|
+      long_build_pr_files.each do |long_build_pr_file|
+        if this_pr_file == long_build_pr_file["file"]
+          conflicts.append("#{tap.default_remote}/pull/#{long_build_pr_file["pr"]}")
+        end
+      end
+    end
+    return unless conflicts.any?
+
+    odie "You are trying to merge a pull request that conflicts " \
+         "with a long running build in #{conflicts.join(",")}"
```

Comment:
Could you show an example error message here?

I think this might also read better:
```suggestion
    odie <<~EOS
      You are trying to merge a pull request that conflicts with
      a long running build in
        #{conflicts * "  \n"}"
    EOS
```

Reply:
Done. I adjusted it and added a comment to the code.